app.rb

```diff
@@ -0,0 +1,170 @@
+require_relative './person'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './rental'
+require_relative './book'
+require_relative './menu_options'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def run
+    menu
+    run_options
+  end
+
+  def list_books
+    print "List of all books:\n\n"
+    if @books.empty?
+      puts "Sorry, no books available at this time.\n"
+      sleep(2)
+      run
+    else
+      @books.each_with_index { |book, idx| print "#{idx + 1}) Title: #{book.title}, Author: #{book.author}\n" }
+      sleep(2)
+    end
+    run
+  end
+
+  def add_book
+    puts 'Please, enter title:'
+    title = gets.chomp
+    puts 'Please, enter author:'
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books << book
+    puts "\n"
+    puts 'Congratulations, you created a book!!!'
+    puts "\n"
+    sleep(2)
+    run
+  end
+
+  def add_person
+    puts 'Please, enter age:'
+    age = gets.chomp.to_i
+    if age < 18
+      puts 'Age is below 18. Does this person has parent permission (y/n)?:'
+      permission = gets.chomp
+      case permission
+      when 'n'
+        puts 'Sorry, it is not possible to create this person.'
+        sleep(2)
+        run
+      when 'y'
+        enroll(age)
+      else
+        puts 'Please, select "y" or "n".'
+        sleep(2)
+        run
+      end
+    else
+      enroll(age)
+    end
+  end
```

Comment:
- [ ] When I'm trying to add person I got this error, kindly fix it.  👌
 
![2](https://user-images.githubusercontent.com/45231276/189264546-949b59fd-5014-47ee-a6fb-93f51059e3ee.PNG)